README.md

```diff
@@ -15,26 +15,22 @@ The C# extension is powered by [OmniSharp](https://github.com/OmniSharp/omnishar
 
 ### **Important!** Breaking Changes as of 1.0.10
 
-* The C# extension now only supports [.NET Core RC2](https://blogs.msdn.microsoft.com/dotnet/2016/05/16/announcing-net-core-rc2/). It no longer supports .NET Core RC1 or ASP .NET 5 RC1.
-* **Support for .csproj projects is partially functional on OS X and Linux.** This will impact anyone doing .csproj development on OS X or Linux (e.g. Unity, Xamarin, etc.). We are working to address this in the near future. However, for now, you can use the [Legacy C# Support extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.omnisharp).
+* The C# extension now only supports [.NET Core RC2](https://blogs.msdn.microsoft.com/dotnet/2016/05/16/announcing-net-core-rc2/) and newer. It no longer supports .NET Core RC1 or ASP .NET 5 RC1.
+* **Support for .csproj projects has been temporarily disabled on OS X and Linux.** This will impact anyone doing .csproj development on OS X or Linux (e.g. Unity, Xamarin, etc.). Rest assured that this will be restored in the near future. However, for now, you can use the [Legacy C# Support extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.omnisharp).
+* _Breaking change for 1.2:_ Debugging .NET Core 1.0 **RC2** on macOS and Linux will no longer work. Please upgrade your project to the released version of 1.0.
 
 ### Found a Bug?
 Please file any issues at https://github.com/OmniSharp/omnisharp-vscode/issues.
 
 ### Debugging
 The C# extension now supports basic debugging capabilities! See http://aka.ms/vscclrdebugger for details.
 
-### What's new in C# extension version 1.1
-
-* Preliminary support for `dotnet test`
-* Fix for OmniSharp installation problems on networks with an http proxy
-* Debugger support for an external console
-* Debugger support for environment variables
-* Support for debugging .NET Core 1.0.0 post RC2 builds
-* Automatic web vs. console debugger configuration detection
-* Detach support
-* Fix expression evaluation errors when referencing assemblies which aren't currently loaded
-* Fix expression evaluation on Windows 7
+### What's new in C# extension version 1.2
+
+* Adds debugger support for new Linux versions -- Ubuntu 16.03, Fedora 23, openSUSE 13.2, and Oracle Linux 7.1
+* Enhanced debug console output - module loads are now output, and there are launch.json options for controlling what is output
+* Support for editing the value of variables in the watch and locals window (requires VS Code 1.3)
```

Comment:
RE: 1.3 - I don't necessarily want to try and encourage people to try the insiders build. It was more of a place holder so that when 1.3 shows up these features would light up. I am happy to just remove the entry if we want for now.